easybuild/easyconfigs/s/Spack/Spack-daint.eb

```diff
@@ -0,0 +1,96 @@
+#@author: Victor Holanda
+easyblock = 'Tarball'
+
+name = 'Spack'
+version = 'daint'
+
```

Comment:
Maybe add a hint in the module name in order to tell users that this version comes from the git master branch?
Something like:
```
versionsuffix = '-git' or '-git-master' 
```